experimental/packages/opentelemetry-sdk-node/src/sdk.ts

```diff
@@ -247,4 +258,16 @@ export class NodeSDK {
         })
     );
   }
+
+  private retrieveSpanProcessors(provider: NodeTracerProvider): SpanProcessor[] | undefined {
```

Comment:
Ditto, if we register the span processors in the `TracerProviderWithEnvExporters.register`, we don't need this method.

Reply:
`retrieveSpanProcessors` has been removed.